Sources/GroundState/py_variational_montecarlo.hpp

```diff
@@ -256,189 +331,84 @@ void AddVariationalMonteCarloModule(py::module &m) {
                 Becca and Sorella (2017), pp. 143-144.
                 https://doi.org/10.1017/9781316417041")EOF");
 
-  py::class_<vmc::Result>(m_vmc, "_VmcResult");
+  m_vmc.def("compute_samples", &ComputeSamples, py::arg{"sampler"},
+            py::arg{"n_samples"}, py::arg{"n_discard"},
+            py::arg{"der_logs"} = py::none(),
+            R"EOF(Runs Monte Carlo sampling using `sampler`.
 
-  m_vmc.def("compute_samples", vmc::ComputeSamples, py::arg("sampler"),
-            py::arg("nsamples"), py::arg("ndiscard") = 0,
-            py::arg("compute_logderivs") = true, R"EOF(
-           Computes a sequence of visible configurations based on Monte Carlo sampling using `sampler`.
+                  First `n_discard` sweeps are discarded. Results of the next
+                  `≈n_samples` sweeps are saved. Since samplers work with
```

Comment:
Maybe write `>=` or `⪆` or similar. `≈` looks like the number of samples could be smaller as well.

Reply:
good point